core/org.osate.pluginsupport/src/org/osate/pluginsupport/ExecuteJavaUtil.java

```diff
@@ -128,7 +128,7 @@ public Method getJavaMethod(String javaMethod) {
 			return obj.getClass();
 		}
 
-		return getJavaClassFromProjects(className);
+		return null;// getJavaClassFromProjects(className);
```

Comment:
Please delete commented out code and now unused method getJavaClassFromProjects.